src/Windows/PnPMQTTWin32-Before/PnPMQTTWin32.cpp

```diff
@@ -0,0 +1,213 @@
+// Access IoTHub via MQTTT Protocol with mosquitto library (no IoT SDK used)
+// Not using PnP Convention 
+//
+
+#include <cstdio>
+#include <fstream>
+#include <thread>
+#include <future>
+
+#include "mosquitto.h"
+
+#include <windows.h>
+
+// CONNECTION information to complete
+#define IOTHUBNAME "[yourIoTHub]"
+#define DEVICEID "[yourDevice]"
+#define PWD "SharedAccessSignature sr=[yourIoTHub].azure-devices.net%2Fdevices%2F[nameofyourdevice]&sig=[tokengeneratedforyourdevice]"
+// You need to generate a SAS Token valid for couple of hours, 
+// use IoT  Hub portal or IoTExplore to generate one
+
+#define CERTIFICATEFILE "IoTHubRootCA_Baltimore.pem"
+
+// computed Host Username and Topic
+#define USERNAME IOTHUBNAME ".azure-devices.net/" DEVICEID "/?api-version=2019-07-01-preview"
+//#define USERNAME IOTHUBNAME ".azure-devices.net/" DEVICEID "/?api-version=2020-09-30"
```

Comment:
I suggest we remove line 25. the old version API is OK for the "before". We'll update it in the "new/ported" code.

Reply:
I've made the change.